frontend/fontlist.lua

```diff
@@ -119,13 +170,64 @@ end
 
 function FontList:getFontList()
     if #self.fontlist > 0 then return self.fontlist end
-    _readList(self.fontlist, self.fontdir)
+
+    local cache_file = DataStorage:getDataDir() .. "/cache/fontinfo.lua"
```

Comment:
I would have said it's sad (although needed) to need to have and maintain a cache for that - because it's painful to code - but you seem to have done that really well, so ok :)